src/main/java/frc/team997/lib/Trajectory/AsymmetricTrapezoidProfile.java

```diff
@@ -0,0 +1,355 @@
+/**
+ Copyright 2022 FRC Team 997
+
+ This program is free software:
+ you can redistribute it and/or modify it under the terms of the
+ GNU General Public License as published by the Free Software Foundation,
+ either version 3 of the License, or (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ See the GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along with SpartanLib2.
+ If not, see <https://www.gnu.org/licenses/>.
+ */
+package frc.team997.lib.Trajectory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
```

Comment:
This is a little wordy tbh- if we want freshmen/ people after we're gone to understand this it should be simplified.

Reply:
what is wordy?